packages/components/select/src/use-select.ts

```diff
@@ -463,7 +463,7 @@ export function useSelect<T extends object>(originalProps: UseSelectProps<T>) {
       return {
         ...props,
         className: slots.innerWrapper({
-          class: clsx(classNames?.innerWrapper, props?.className),
+          class: clsx(classNames?.innerWrapper, !label && "pt-0", props?.className),
```

Comment:
Hey @ChiuMungZitAlexander we cannot add styles inside the components logic/hooks because tailwindcss will not take them

All the styles should be added into the `theme/components` packages

Reply:
You could use a data property `data-has-label` check the use-select code